packages/bot-avs/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@wireapp/bot-avs",
```

Comment:
Please add a `description` field which explains the features of this bot.

Reply:
https://github.com/wireapp/wire-web-packages/pull/3070/commits/c1b88bb85d9d103c5380c3618907e670e1f58d7a